tools/gstf_preparation/gstf_preparation.py

```diff
@@ -92,6 +92,12 @@ def create_tables(conn):
         FROM transcript JOIN gene
         USING (gene_id)''')
 
+    cur.execute('''CREATE TABLE syntenic_region (
+        syntenic_region_id VARCHAR NOT NULL,
```

Comment:
Would it make sense to just store `seq_region_name` in the table instead of a numeric `syntenic_region_id`, since in the current implementation this is per-species, so not a unique id?

Reply:
yes, agree I will change it..